src/libcpu/src/jit/jit_float.cpp

```diff
@@ -37,14 +37,14 @@ hostHasFMA3()
 }
 
 static void
-truncateToSingleSd(PPCEmuAssembler& a, const PPCEmuAssembler::XmmRegister& reg)
```

Comment:
"round" = round according to FPSCR rounding mode, "truncate" = discard low bits (i.e. round toward zero). The behavior is actually different for ps0 and ps1 -- see my tests.